src/browser/modules/Stream/SysInfoFrame/SysInfoFrame.tsx

```diff
@@ -81,20 +83,64 @@ export class SysInfoFrame extends Component<
     idAllocation: [],
     pageCache: [],
     transactions: [],
-    error: '',
+    errorMessage: null,
     results: false,
-    success: false,
     autoRefresh: false,
-    autoRefreshInterval: 20 // seconds
+    autoRefreshInterval: 20, // seconds
+    namespacesEnabled: false,
+    userConfiguredPrefix: 'neo4j'
   }
-  helpers = this.props.hasMultiDbSupport ? helpers : legacyHelpers
 
   componentDidMount(): void {
-    this.getSysInfo()
+    this.getSettings()
+      .then(this.getSysInfo.bind(this))
+      .catch(() => this.setState({ errorMessage: "Couldn't fetch settings" }))
   }
 
+  getSettings = (): Promise<void> =>
+    new Promise((resolve, reject) => {
+      const { bus, isConnected } = this.props
+
+      if (bus && isConnected) {
+        bus.self(
+          CYPHER_REQUEST,
+          {
+            query: 'CALL dbms.listConfig("metrics.")',
+            queryType: NEO4J_BROWSER_USER_ACTION_QUERY
+          },
+          ({ success, result }) => {
+            if (success) {
+              const newState = result.records.reduce(
+                (newState: Partial<SysInfoFrameState>, record: any) => {
+                  const name = record.get('name')
+                  const value = record.get('value')
+                  if (name === 'metrics.prefix') {
+                    return { ...newState, userConfiguredPrefix: value }
+                  }
+
+                  if (name === 'metrics.namespaces.enabled') {
+                    return { ...newState, namespacesEnabled: value === 'true' }
+                  }
+
+                  return newState
+                },
+                {}
+              )
+
+              this.setState(newState)
+              resolve()
+            } else {
+              reject()
```

Comment:
maybe return the "failed to run listConfig" message here, and a different error when not connected?